src/Contrib/Otlp/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "open-telemetry/exporter-otlp-common",
+  "description": "OTLP common code for OpenTelemetry exporters.",
```

Comment:
I've just realised that this is also where otel metrics exporter lives. This will need some thought and probably moving things around.

Reply:
on reflection, I think the metrics stuff does belong here and it's not a problem.